ReleaseNotes.md

```diff
@@ -1,5 +1,16 @@
 # Release Notes
 
+## 0.11.0
+
+* **Potentially Breaking** `InvariantConvert` and `StringBuilderUtility.AppendInvariant` have different behavior for floating-point numbers.
+  * The `G17` (or `G9`) format is used (instead of `R`) to guarantee that floating-point numbers round-trip correctly. This may cause numbers to be serialized differently than in previous versions of Libronix.Utility.
```

Comment:
```suggestion
  * The `G17` (or `G9`) format is used (instead of `R`) to guarantee that floating-point numbers round-trip correctly. This may cause numbers to be serialized differently than in previous versions of Faithlife.Utility.
```